sites/website/build-homepage.js

```diff
@@ -0,0 +1,39 @@
+// FIXME This file is a temporary script and should be obsoleted asap
```

Comment:
When FIXME or TODOs are added I have in the past I've found useful to include an issue number just in case.  I've done something like "FIXME: 1222 Temporary script to be removed asap" for example.